python_modules/dagster/dagster/_core/host_representation/repository_location.py

```diff
@@ -462,6 +464,10 @@ def get_external_schedule_execution_data(
             if scheduled_execution_time
             else None,
         )
+        if isinstance(result, ExternalScheduleExecutionErrorData):
+            raise DagsterUserCodeProcessError.from_error_info(result.error)
```

Comment:
error's a bit odd since its in the same process, but in-process is essentially for testing anyway